Jellyfin.Plugin.Webhook/Destinations/BaseOption.cs

```diff
@@ -12,6 +12,26 @@ namespace Jellyfin.Plugin.Webhook.Destinations;
 /// </summary>
 public abstract class BaseOption
 {
+    /// <summary>
+    /// The data fields that have historically been manually escaped in <see cref="DataObjectHelpers"/>.
+    /// More fields may be added as needed.
+    /// </summary>
+    private readonly List<string> fieldsToEscape =
```

Comment:
```suggestion
    private static readonly string[] FieldsToEscape =
```

No reason for this to be a list